stories/clickAnalytics.stories.js

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from '@storybook/html';
+import { withHits } from '../.storybook/decorators';
+
+storiesOf('ClickAnalytics', module).add(
+  'basic',
```

Comment:
can you give a hogan story too, or is that not finished yet?

Reply:
if we want to be able to use it in hogan, we'd need to a specific `clickedObjectIDsAfterSearchHogan`.
It's not impossible it's just not pretty.

```js
        analytics: {
          clickedObjectIDsAfterSearch: eventName => {
            const payload = {
              eventName,
              index,
              queryID,
              objectIDs: [hit.objectID],
              positions: [hitsPerPage * page + position + 1],
            };
            return `
              data-analytics-event="clickedObjectIDsAfterSearch"
              data-analytics-payload='${JSON.stringify(payload)}'
            `;
          },
          clickedObjectIDsAfterSearchHogan: () => {
            return eventName => {
              const payload = {
                eventName,
                index,
                queryID,
                objectIDs: [hit.objectID],
                positions: [hitsPerPage * page + position + 1],
              };
              return `
              data-analytics-event="clickedObjectIDsAfterSearch"
              data-analytics-payload='${JSON.stringify(payload)}'
            `;
            };
          },
```
Would it be fine to enforce the use of function templates in clickAnalytics case?

Reply:
Seeing as the difference is only a function call, what about making the regular one `clickedObjectIDsAfterSearch()`

Reply:
ok, so we'd be using it in the following way.
```html
<button ${clickedObjectIDsAfterSearch()('Add to cart'))}>
   Add to basket
</button>
```
```handlebars
<button {{#analytics.clickedObjectIDsAfterSearch}}Add to cart{{/analytics.clickedObjectIDsAfterSearch}}>
   Add to basket
</button>
```
But I'm not sure it's ideal.